openpype/hosts/nuke/plugins/publish/extract_slate_frame.py

```diff
@@ -242,7 +242,7 @@ def _render_slate_to_sequence(self, instance):
 
         # render slate as sequence frame
         nuke.execute(
-            instance.data["name"],
+            str(instance.data["name"]),
```

Comment:
Is there a reason to convert it to string?

Reply:
I can't reproduce now, but some combination of project settings failed without the conversion.

Reply:
Yes please. If the name is `None` it should crash :)

Reply:
can we revert this change to previous version, please